.github/workflows/ci.yml

```diff
@@ -20,13 +20,13 @@ jobs:
         python-version: 3.10.7
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip
+        python -m pip install --upgrade pip==24.0
         pip install -r requirements.txt
         pip install pytest pytest-cov pytest-shell
         python setup.py install
     - name: Install Python Poetry
       run: |
-        python -m pip install --upgrade pip
```

Comment:
again, why 24.0?

Reply:
Will remove that again. Thought this was the issue the ci was failing since it was the only thing it changed since it ran the last time.